imports/ui/components/T9n.js

```diff
@@ -25,7 +26,24 @@ Meteor.startup(function() {
     med: "Medicina",
     pedag: "Pedagogia",
     psicol: "Psicologia",
-    pubpr: "Publicidade e Propaganda"
+    pubpr: "Publicidade e Propaganda",
+    //CATEGORIES
+    "advog-lei": "Advogados & Leis",
```

Comment:
We can only use json keys with the symbol **-** with quotes

An alternative is using the symbol **_** instead